assets/src/edit-story/app/media/utils/useFFmpeg.js

```diff
@@ -236,8 +240,10 @@ function useFFmpeg() {
   );
 
   return {
-    isFeatureEnabled,
-    isTranscodingEnabled,
+    isTranscodingEnabled:
+      hasUploadMediaAction &&
+      isTranscodingEnabled &&
+      isCrossOriginIsolationEnabled,
```

Comment:
I find this a little hard to read. Can we set it to a const for readability.

Reply:
Done